app/views/catalog/_show_upper_metadata_default.html.erb

```diff
@@ -1,6 +1,26 @@
 <% if blacklight_config.show.component_metadata_partials.present? %>
   <% parents = Arclight::Parents.from_solr_document(document).as_parents %>
 
+  <% if document.restricted? %>
```

Comment:
This should also display if `document.review?` I think. The example in the ticket is https://findingaids-qa.princeton.edu/catalog/AC137_c0109, which has "restrictions may apply" instead of "restricted"

Reply:
Logic to handle the `document.review?` scenario has been added.